pkg/apis/serving/k8s_validation_test.go

```diff
@@ -543,6 +550,8 @@ func TestPodSpecMultiContainerValidation(t *testing.T) {
 }
 
 func TestPodSpecFeatureValidation(t *testing.T) {
+	runtimeClassName := "test"
```

Comment:
I assume there is no good way to validate the values passed (besides podspec dry run?), since it varies so much vendor to vendor...

Reply:
I suppose the controller could validate that the RuntimeClass with that name exists and add to the knative service's status and emit an event if it doesn't. Currently the controller will fail to create pods and users will see that the service didn't create and see the detailed reason in events already so I'm not sure it would be a much better UX.